perfkitbenchmarker/vm_util.py

```diff
@@ -246,10 +246,10 @@ def WrappedFunction(*args, **kwargs):
           sleep_time = poll_interval * fuzz_multiplier
           if ((time.time() + sleep_time) >= deadline or
               (max_retries >= 0 and tries > max_retries)):
-            raise e
+            raise
```

Comment:
Sorry, I am interested because I see this pattern being used in spanner work, and I thought 'e' was the original exception. Is that not the case. Will 'e' and the original exception from the function being retried ever differ ? If so I may have to request some changes elsewhere. Thanks for the link